evaporate.js

```diff
@@ -1198,7 +1211,11 @@
                         authRequester.onFailedAuth(err);
                         return;
                     }
-                    authRequester.auth = JSON.parse(data.Payload);
+                    var payload = JSON.parse(data.Payload);
+                    if (typeof con.signResponseHandler === 'function') {
+                      payload = con.signResponseHandler(payload) || payload;
+                    }
```

Comment:
Are you sure this duplication is correct? You've got essentially the same code as in lines 1152-1154.